spec/std/isa/ext/Sdtrig.yaml

```
@@ -40,6 +40,7 @@ params:
     schema:
       type: string
       enum: [rw, read-only-0, read-only-1]
+      default: "rw"
```

I see the CI failure just now. Could you try removing the quotation marks here? Since it's an enum, maybe providing a string instead of the enum value is causing the issue.
```suggestion
      default: rw
```
Here and below.

---

Thanks, Done!